gst/nnstreamer/tensor_transform/tensor_transform.c

```diff
@@ -1478,55 +1478,37 @@ gst_tensor_transform_convert_dimension (GstTensorTransform * filter,
         int b = filter->data_dimchg.to;
 
         for (i = 0; i < NNS_TENSOR_RANK_LIMIT; i++) {
-          if (i < a && i < b) {
+          if ((i < a && i < b) || (i > a && i > b) || a == b) {
             out_info->dimension[i] = in_info->dimension[i];
-          } else if (i > a && i > b) {
-            out_info->dimension[i] = in_info->dimension[i];
-          } else if (a > b) {
-            if (i == b) {
-              out_info->dimension[i] = in_info->dimension[a];
-            } else {
+          } else if (i == b) {
+            out_info->dimension[i] = in_info->dimension[a];
+          } else {
+	    if (a > b) {
```

Comment:
indent? BTW, about about using `else if`

Reply:
Thank you for your support! Let me reflect your review.